engine/doc/engine-doc.md

```diff
@@ -30,3 +30,20 @@ The main main job is done by whitelist class, it parses file and compares final
 This class is a singleton witch is replace by another instance each time conf is reloaded.
 
 Checkable class inherited by service and host classes keeps the last result of whitelist's check in cache in order to reduce CPU whitelist usage.
+
+## extended configuration
+Users can pass an additional configuration file to engine. Gorgone is not aware of this file, so users can override centengine.cfg configuration.
+Each entry found in additional json configuration file override his twin in centengine.cfg
+
+### example of command line
+/usr/sbin/centengine --config-file=/tmp/centengine_extend.json /etc/centreon-engine/centengine.cfg
```

Comment:
```suggestion
### Example of command line
```
/usr/sbin/centengine --config-file=/tmp/centengine_extend.json /etc/centreon-engine/centengine.cfg
```

Reply:
I remember we can also specify several files, something like
```
/usr/sbin/centengine -c file1.json -c file2.json /etc/centreon-engine/centengine.cfg
```
If files1 and file2 share the same item `foo`, which one has the last word?
I think you should also specify this case in the doc.